src/genophenocorr/analysis/predicate/genotype/_geno_bool.py

```diff
@@ -282,4 +283,56 @@ def __str__(self):
         return repr(self)
 
     def __repr__(self):
-        return f'ProtFeaturePredicate(tx_id={self._tx_id}, exon_number={self._pf_name})'
+        return f'ProtFeaturePredicate(tx_id={self._tx_id}, protein_feature_name={self._pf_name})'
+
+
+class ProtRegionPredicate(GenotypeBooleanPredicate):
+    """
+    `ProtRegionPredicate` tests if the `patient` has a variant that overlaps with a given region of the protein.
```

Comment:
A small note - there is no `patient` in the scope of this comment (there is one in the `test` method). Therefore, it is probably better to remove the back ticks.